Training25/Program.cs

```diff
@@ -3,11 +3,37 @@
 // Copyright (c) Metamation India.
 // ------------------------------------------------------------------
 // Program.cs
-// Program on main branch.
+// Program on T1 branch.
+// Program to convert decimal input to binary and hexadecimal.
 // ------------------------------------------------------------------------------------------------
+using System.Text;
+using static System.Console;
 namespace Training25;
+
 internal class Program {
    static void Main (string[] args) {
-      Console.WriteLine ("Hello, World!");
+      do {
+         Clear ();
+         Write ("Input: ");
+         if (int.TryParse (ReadLine (), out int num) && num >= 0) {
+            WriteLine ("BIN: " + ConvertToBase (2, num));
+            WriteLine ("HEX: " + ConvertToBase (16, num));

```

Comment:
```suggestion
            WriteLine ($"BIN: {ConvertToBase (2, num)}\nHEX: {ConvertToBase (16, num)}"});
```
FYI, this would work the same.
Look up Interpolated Strings in C#